data/phone/text/irwin_caller.asm

```diff
@@ -240,15 +252,15 @@ UnknownText_0x647d8:
 	para "You so rock!"
 	done
 
-UnknownText_0x64846:
+IrwinEscapadeText:
```

Comment:
`IrwinYourEscapadesRockText`? Again, if you're paraphrasing text to make the labels more descriptive (like `IrwinRocketRumorText`) then avoid disconnected noun series.